### PR TITLE
Continue after logging Twilio exception to HoneyBadger.

### PR DESCRIPTION
**Pull request by ashercodeorg**
For more information, see https://app.honeybadger.io/projects/45435/faults/31181878.

### CODE REVIEW

File: bin/cron/delete_twilio_data [hunk 1]
```diff
@@ -44,10 +44,9 @@ def main
 
         message.delete
       rescue Twilio::REST::RequestError => e
-        # TODO(asher): Consider whether we should log the exception and continue to
-        # attempt to delete further messages rather than stopping deletion.
         sanitized_error_message = e.message.sub CDO.twilio_sid, 'CDO.twilio_sid'
-        raise "Twilio::REST::RequestError (#{message.status}): #{sanitized_error_message}"
+        HipChat.message 'infra-twilio', sanitized_error_message, color: 'red'
```
Comment (ashercodeorg) [inline bin/cron/delete_twilio_data]: Note that I intend to manually monitor this room.

**Comment by wjordan** [comment]
Is there a specific reason that I'm not aware of for moving these specific messages into Slack rather than leaving them in the existing HoneyBadger report? Entirely your preference if this is how you want to monitor this particular error, but my initial reaction is that this seems like a very strange choice, considering HoneyBadger is a specialized tool for logging and aggregating application error messages like this one, and I don't notice anything particularly unusual about this error that can't be handled by it.

**Comment by ashercodeorg** [comment]
Your initial reaction is sound. Mostly, I was wanting to allow the loop to continue with `next`. Instead, I should rescue the exception, manually send an HB report, and continue. Expect another commit at some point.

**Comment by ashercodeorg** [comment]
PTAL.

**Pull request merged**